sdks/go/pkg/beam/io/filesystem/filesystem.go

```diff
@@ -92,6 +93,12 @@ type Interface interface {
 // The following interfaces are optional for the filesystems, but
 // to support
```

Comment:
I think the cut off sentence here should say 
`// to support more efficient or composite operations when possible.`

eg. Renamer is for atomic operations, instead of a copy + a remove.

Reply:
Thanks, updated!